AfricanReligions/events/afr_misc_events.txt

```diff
@@ -235,4 +235,219 @@ character_event = {
             }
         }
     }
+}
+
+# Bori Selecting Patron Deities events
+character_event = {
+    id = AFRREL.00500
+    desc = EVTDESC_AFRREL00500
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_desert
+
+    is_triggered_only = yes
+
+    immediate = {
+        if = {
+            limit = { if_playable = yes }
+            any_courtier = {
+                limit = {
+                    prisoner = no
+                    religion = root
+                    age = 16
+                    has_minor_deity_trigger = no
+                }
+                character_event = { id = AFRREL.00500 days = 1 }
+            }
+        }
+    }
+
+    option = {
+        name = EVTOPTA_AFRREL00501
```

Comment:
You can name the localization keys anything, but I think in general we should name them so that we know what they're for later.  So, the first option name of event 00500 would be EVTOPTA_AFRREL00500, and the next one would be EVTOPTB_AFRREL00500, etc.  The ones ending in AFRREL00501 would then be fore event 00501 if it exists, and so forth.

Reply:
What do you mean because I think this is what I've done already?

Reply:
Your key says EVTOPTA_AFRREL00501, which indicates that it's the first option on event 00501, but it's actually the first option on event 00500.  The second option is EVTOPTA_AFRREL00502, which indicates that it's the first option on event 00502, but actually it's the second option on event 00500.